en/docs/deploy/deploy-api/grouping-apis-with-vhosts.md

```diff
@@ -0,0 +1,142 @@
+
+3.  Add a new Gateway Environment.
+
+     1. Click **Gateways**, and then click **Add Gateway Environment**.
+
+         [![Menu to add Microgateway label]({{base_path}}/assets/img/learn/add-gateway-environment-menu.png)]({{base_path}}/assets/img/learn/add-gateway-environment-menu.png)
+
+     2. Enter a name, display name, description, and a virtual host.
+
+         The virtual hosts will define each of the custom hostnames.
+
+          | **Environment** | **Display Name** | **Description**                               | **Virtual Host** |
+          |-------------|--------------|-------------------------------------------|--------------|
+          | us-region   | US Region    | Gateway environment deployed in the US region. | us.wso2.com  |
+
+          <a href="{{base_path}}/assets/img/learn/add-gateway-environment.png">
+              <img src="{{base_path}}/assets/img/learn/add-gateway-environment.png" alt="Add a Gateway Environment"
+              title="Add a Gateway Environment" width="500px" />
+          </a>
+
+          Add another virtual host `foods.com` by clicking **New VHost** and click **Save** to save the environment.
+
+### Create an environment using the configuration file
+
+1. Open to the `<APIM-HOME>/repository/conf/deployment.toml` file.
+
+2. Create a Gateway environment.
+
+     Create a Gateway environment with the following Gateway configurations, which include `us.wso2.com` and `foods.com` as the VHosts.
+
+    !!! note
+
+        - When the WSO2 API Manager server is running, the Gateway environments that you added via the `deployment.toml` configuration file are displayed in the Gateway Environments list page in the Admin Portal in read-only mode. 
+        - Therefore, ensure to add the VHosts that correspond to the Gateway at the time of adding the Gateway environment itself.
+
+    ```toml
+    [[apim.gateway.environment]]
+    name = "us-region"
+    display_name = "US Region"
+    type = "hybrid"
+    display_in_api_console = true
+    description = "Gateway environment deployed in the US region."
+    show_as_token_endpoint_url = true
+    service_url = "https://localhost:${mgt.transport.https.port}/services/"
+    username= "${admin.username}"
+    password= "${admin.password}"
+    ws_endpoint = "ws://localhost:9099"
+    wss_endpoint = "wss://localhost:8099"
+    http_endpoint = "http://localhost:${http.nio.port}/gateway"
+    https_endpoint = "https://localhost:${https.nio.port}/gateway"
+
+    [[apim.gateway.environment.virtual_host]]
+    ws_endpoint = "ws://us.wso2.com:9099"
+    wss_endpoint = "wss://us.wso2.com:8099"
+    http_endpoint = "http://us.wso2.com/gateway"
+    https_endpoint = "https://us.wso2.com/gateway"
+
+    [[apim.gateway.environment.virtual_host]]
+    ws_endpoint = "ws://foods.com:9099"
+    wss_endpoint = "wss://foods.com:8099"
+    http_endpoint = "http://foods.com/gateway"
+    https_endpoint = "https://foods.com/gateway"
```

Comment:
Please add these configurations in the Config catalog if it has not been added yet.

Reply:
Updated config catalog